src/App.js

```diff
@@ -0,0 +1,34 @@
+import { Routes, Route } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { fetchMissions } from './redux/features/missionsSlice';
+import { fetchData } from './redux/features/rocketsSlice';
+import Navigation from './components/navigation';
+import NotFound from './components/notFound';
+import Profile from './components/Profile';
+import Missions from './components/Missions';
+import './styles/general.css';
+import './styles/profile.css';
+import RocketsDisplay from './components/rocketsDisplay';
+
+function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchData());
+    dispatch(fetchMissions());
+  }, []);
```

Comment:
- [ ] Rockets is the default view, so its data would be fetched when the application starts. However, the missions data should only be fetched (once) when a user navigates to the Missions section. Kindly fix this.